bootstrap/v2/pkg/kfapp/existing_arrikto/existing_test.go

```diff
@@ -47,3 +51,170 @@ func TestGenerateCert(t *testing.T) {
 		})
 	}
 }
+
+func TestGetEndpoints(t *testing.T) {
+	cases := []struct {
+		name                     string
+		kubeflowEndpoint         string
+		oidcEndpoint             string
+		expectedKubeflowEndpoint string
+		expectedOIDCEndpoint     string
+		expectError              bool
+	}{
+		{
+			name:                     "ip",
+			kubeflowEndpoint:         "https://172.56.12.125",
+			expectedKubeflowEndpoint: "https://172.56.12.125",
+			expectedOIDCEndpoint:     "https://172.56.12.125:5556/dex",
+			expectError:              false,
+		},
+		{
+			name:                     "hostname",
+			kubeflowEndpoint:         "https://example.com",
```

Comment:
just want to get more background here? What's the kubeflow endpoint? load balancer endpoint for kubeflow centraldashboard?

Reply:
The Kubeflow endpoint is the address of the Istio Ingress Gateway.
In our case, that is the address of the LoadBalancer service `istio-ingressgateway`.